src/Node/ChildProcess.purs

```diff
@@ -263,6 +266,9 @@ type SpawnOptions =
   { cwd :: Maybe String
   , stdio :: Array (Maybe StdIOBehaviour)
   , env :: Maybe (Object String)
+  , shell :: Maybe String
+  , timeout :: Maybe Number
```

Comment:
The types for these probably aren't quite ideal - I'd suggest using `Milliseconds` for `timeout`, and `shell` perhaps needs a new sum type that covers the `true` argument. I've actually never used anything but `true`, thought it was a mistake here until I checked the node docs!